_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -31,7 +31,7 @@ This rule applies to each [link history state][] of any [semantic link][], for w
 
 ## Expectation
 
-For each test target, there exists at least one [adequate inspection state][] and at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**.
+For each target, when its [semantic link][] is not in the [focused][] state and the [hovered][] state at the same time, there exists at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**.
```

Comment:
I understand that as different from the previous version…
"there exist at least one adequate state [where this is good]" means we pass if this is good in (not H, not F) OR in (H, not F) OR in (not H, F).
"when it is not H and F at the same time", first, it is less clear that we need to check the various inspection states. Second, I understand it as needed to be good in all of (not H, not F) AND (H, not F) AND (not H, F) states.

Reply:
I agree. This seems to imply that it needs to be true in all of those states. 
I'm backtracking a bit in the expectation, accordingly.